Application/Source/Components/Registration/RegistrationRootView.ui.xml

```diff
@@ -8,50 +8,50 @@
     extend="all"
     rootView="true">
 
-    <styles name="ReactantStyles"></styles>
+<styles name="ReactantStyles"></styles>
 
-    <ImageView
-    image="blurred-splash"
-    style=":general:splash"
-    layout:edges="super"/>
+<ImageView
```

Comment:
`Component` is considered to be the highest level, which means these should be indented. The body of the components should also be on the next indentation level compared to the component being defined.

That would make this `ImageView`:
```
<ImageView
    image="blurred-splash"
    style=":general:splash"
    layout:edges="super" />
```
Which as a whole is also indented by one level because it's inside the whole `Component`.